src/ActionHistoryList.js

```diff
@@ -0,0 +1,31 @@
+import React, {PropTypes} from 'react';
+import Action from './Action';
+
+const ActionHistoryList = props => {
+  return (
+    <div className="action-history-list">
+      <h3>Action History</h3>
+      {props.actions.length > 0 ? null : 'Nothing has happened yet!'}
+      <div className="actions-list">
+        <ul>
+          {props.actions.slice(0, 10).map(action => {
+            return (
+              <Action
+                key={action._id}
+                action={action}
```

Comment:
Inside of Action.js you're trying to access the actionMessage property of action, so make sure you're actually passing in an object. I think right now you're passing a string value.